package.json

```diff
@@ -12,9 +12,11 @@
     "lint": "eslint **/*.{js,jsx,ts,tsx} --quiet",
     "test": "jest",
     "test:update": "jest -u",
-    "prepare": "husky"
+    "prepare": "husky",
+    "msw:init": "msw init public/ --save"
```

Comment:
Could you add some blurb into the documentation about each of the scripts? (you'll need to rebase first though as I updated the documentation structure slightly)

Reply:
We should probably add the `npx pa11y-ci` into here too and have it run as part of yarn test 🤔